pageserver/src/walingest.rs

```diff
@@ -2079,4 +2079,66 @@ mod tests {
 
         Ok(())
     }
+
+    /// Replay a wal segment file taken directly from safekeepers.
+    ///
+    /// This test is useful for benchmarking since it allows us to profile only
+    /// the walingest code in a single-threaded executor, and iterate more quickly
+    /// without waiting for unrelated steps.
+    #[tokio::test]
+    async fn test_ingest_real_wal() -> anyhow::Result<()> {
```

Comment:
assuming this test would start to fail it would be much nicer to understand where it failed if this was a plain unit `()` returning test because then you'd have the line information whereas if this fails, you need to configure logging and then hope to catch it again.

So suggest removing the return type and just doing `unwrap` or `expect` where you'd `?`.